### DISCUSSION TITLE
Is there any steps to migrate an existing Terraform code base with that of the Gruntwork Modules

### DISCUSSION THREAD

**Discussion by vramanufidato** [gruntwork-io/knowledge-base#587] [How do I] [answered]
Hi Team, 

I was referring to the Gruntwork documentation, but i couldn't get any documentation / steps on how to migrate an existing terraform codebase by using the Gruntwork modules.

I happen to check the readme section of below modules.
[terragrunt-infrastructure-modules-example](https://github.com/gruntwork-io/terragrunt-infrastructure-modules-example)
[terragrunt-infrastructure-live-example](https://github.com/gruntwork-io/terragrunt-infrastructure-live-example)

If there is any documentation maintained in [https://docs.gruntwork.io](https://docs.gruntwork.io/reference/services/intro/overview)
will be helpful by referring to an example and a repository for the same.

Thanks in advance

---

<ins datetime="2022-11-09T08:15:43Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109574">Tracked in ticket #109574</a></p>
</ins>

**Comment by ebeneliason** [comment, marked as answer]
This is a complex question. Our consulting offering may be a good way forward for you if you already have a Gruntwork subscription. On the other hand, if you're not a subscriber and are evaluating whether Gruntwork could work for your needs, our sales team can help with that as well. I encourage you to contact sales@gruntwork.io as a next step and reference this discussion.
